src/uniprotkb/components/entry/NamesAndTaxonomySection.tsx

```diff
@@ -34,6 +35,19 @@ const NamesAndTaxonomySection = ({ data, primaryAccession }: Props) => {
     >
       <h3>Protein names</h3>
       <ProteinNamesView proteinNames={data.proteinNamesData} />
+      {!!domains?.length && (
+        <>
+          <h5>{`Including ${domains.length} ${pluralise(
```

Comment:
This needs to be an h4 if it follows an h3. If you need to make it the size of an h5, you can use the `tiny` class from franklin

Reply:
👍  coincidentally looks better too